.github/workflows/pkgdown.yaml

```diff
@@ -32,6 +32,7 @@ jobs:
 
       - name: Install dependencies
         run: |
+          brew imagemagick@6
           install.packages("remotes")
           remotes::install_deps(dependencies = TRUE)
```

Comment:
`remotes::install_deps(dependencies = TRUE)` will also install the `Suggests` packages.